### ISSUE TITLE
Confidence values above 1 for rtmpose model

### ISSUE THREAD

**Issue by philipqueen** [DeepLabCut/DeepLabCut#3126]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Operating System

Mac OS Sequoia 15.6.1

### DeepLabCut version

3.0.0rc10

### What engine are you using?

pytorch

### DeepLabCut mode

single animal

### Device type

Builtin GPU (metal)

### Bug description 🐛

Confidence values are not constrained between 0 and 1. For example, on one model the confidence values are mostly between 2 and 4, and on another they mostly stay below 1 but spike to over 12. My expectation is that these values represent a range from 0 (not at all confident) to 1 (100% confident), as is the case for the standard resnet model.

Here are some likelihood plots showing the issue:

<img width="646" height="227" alt="Image" src="https://github.com/user-attachments/assets/b5cabcab-28d9-446c-a617-b7108fd550f8" />

<img width="811" height="299" alt="Image" src="https://github.com/user-attachments/assets/28fa6332-275f-41fb-bffe-8e31fa54a77f" />

### Steps To Reproduce

1. Create a new environment with `deeplabcut==3.0.0rc10`
2. Create a new project and label data
3. Create the training dataset with an rtmpose model, like `deeplabcut.create_training_dataset(config_path, net_type="rtmpose_x")`
4. Train the model and analyze the data

### Relevant log output

```shell

```

### Anything else?

The mm pose implementation has a similar issue (with one potential solution in the comments): https://github.com/open-mmlab/mmpose/issues/2755
This article explains where the confidence scores are derived from in the rtmpose architecture: https://deepwiki.com/Tau-J/rtmlib/4.2.1-rtmpose#simcc-postprocessing

The softmax solution in the first file *seems* correct, but I haven't dug into the guts of the deeplabcut implementation to see where to apply it.

EDIT:
It appears there is a bool to apply the softmax function in the `SimCCPredictor`, and an option to normalize. Looking at my config, it's possible the `normalize: false` line is the issue here:
```
  heads:
    bodypart:
      type: RTMCCHead
      weight_init: RTMPose
      target_generator:
        type: SimCCGenerator
        input_size:
        - 384
        - 384
        smoothing_type: gaussian
        sigma:
        - 6.93
        - 6.93
        simcc_split_ratio: 2.0
        label_smooth_weight: 0.0
        normalize: false
      criterion:
        x:
          type: KLDiscreteLoss
          use_target_weight: true
          beta: 10.0
          label_softmax: true
        y:
          type: KLDiscreteLoss
          use_target_weight: true
          beta: 10.0
          label_softmax: true
      predictor:
        type: SimCCPredictor
        simcc_split_ratio: 2.0
```

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by AlexEMG** [comment]
This is a known bug in the version you use. We fixed it here: https://github.com/DeepLabCut/DeepLabCut/pull/3078 --> Can you check the latest version: https://github.com/DeepLabCut/DeepLabCut/commit/afcf8b41ea9279dde6e83267578666ed8d73c4ff (rc13!)

**Comment by philipqueen** [comment]
Great, thank you! The latest release showing in the github releases is rc10, so I thought I was up to date. I appreciate the help.
